.github/dependabot.yml

```
@@ -26,31 +26,56 @@ updates:
     package-ecosystem: npm
     schedule:
       interval: monthly
```

Could be useful the default reviewer?

```suggestion
    schedule:
      interval: monthly
    reviewers:
      - "northerntechhq/frontend-dependabot-reviewers"
```